Kudu.ContainerServices.Agent/Security/DeploymentCredentialsAuthenticationOptions.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Authentication;
+
+namespace Kudu.ContainerServices.Agent.Security
+{
+    public class DeploymentCredentialsAuthenticationOptions : AuthenticationSchemeOptions
```

Comment:
why is this class needed ?

Reply:
Not needed; was added temporarily. Will remove.